### PR TITLE
unset keyword and a fix for and_ff

### PR DESCRIPTION
**Pull request by romainmenke**
- https://caniuse.com/?search=unset
- https://developer.mozilla.org/en-US/docs/Web/CSS/unset

--------

`and_ff` had the same issue as `and_chr`

### CODE REVIEW

File: cssdb.settings.json [hunk 1]
```diff
@@ -983,6 +983,25 @@
...
+    "docs": {
+      "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/unset"
+    },
+    "example": "div {\n  border-color: unset;\n  color: unset;\n}",
+    "polyfills": [
+      {
+        "type": "PostCSS Plugin",
+        "link": "https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-unset-value"
+      }
+    ]
```
Comment (romainmenke) [inline cssdb.settings.json:1003]:
```suggestion
    "example": "div {\n  border-color: unset;\n  color: unset;\n}"
```

Not released yet, but wanted it visible somewhere on github

**Review by Antonio-Laguna — APPROVED** [review]
Thanks for this! Should we just add the plugin too?

**Comment by romainmenke** [comment]
> Thanks for this! Should we just add the plugin too?

I didn't want to create a rush by adding the plugin here too.
But we can review that one first and merge it?

Then we can update cssdb and release the plugin quickly after each other with confidence?

**Comment by Antonio-Laguna** [comment]
You're absolutely right, I'll just merge this and release !

**Pull request merged**